src/pages/Delegate/index.tsx

```diff
@@ -105,7 +143,17 @@ export const Delegate = () => {
       <Label className="flex">
         Conviction:<div className="ml-2">{convictionShow}</div>
       </Label>
-      <Button onClick={onSign} disabled={amount === 0}>
+
+      {amount === 0 && (
+        <AlertNote
+          title={msgs.zeroAmount.title}
+          message={msgs.zeroAmount.message}
+        />
+      )}
+      <Button
+        onClick={onSign}
+        disabled={amount === 0 || !api || !selectedAccount}
```

Comment:
This is still very week and I bet things break if we put the wrong comma, or some string in that field, But anyway, trying to tackle more cases.

```suggestion
      {amount <= 0 && (
        <AlertNote
          title={msgs.zeroAmount.title}
          message={msgs.zeroAmount.message}
        />
      )}
      <Button
        onClick={onSign}
        disabled={amount <= 0 || !api || !selectedAccount}
```

Reply:
I have fixed that, by ensuring that commas are correctly replaced and all amounts are correctly fixed based on the decimals of the chain